### ISSUE TITLE
allyesconfig does not build

### ISSUE THREAD

**Issue by mpe** [linuxppc/linux#1]
```
$ make allyesconfig ; make
...
e1000_hw.c:(.text+0x1ff72c8): relocation truncated to fit: R_PPC64_REL24 against symbol '_savegpr0_30' defined in .text.save.restore section in arch/powerpc/lib/built-in.o
```

And so on.

sfr is working on a solution using thin archives.

**Comment by npiggin** [comment]
$ ls -sh vmlinux
303M vmlinux

**Comment by mpe** [comment]
Nice! Will be interesting to see if we can boot it :)

**Comment by npiggin** [comment]
Fixed.

http://kisskb.ellerman.id.au/kisskb/target/2654/

**Comment by mpe** [comment]
Implemented by @npiggin in 43c9127d94d62a232ed33ed2eab8a08657ce5472 and a5967db9af51a84f5e181600954714a9e4c69f1f.

Thanks @npiggin !! :clap: :boom: :tiger2:
